README.md

```diff
@@ -1,4 +1,47 @@
-# JanusGraph Python driver
+# Python Client drivers for JanusGraph
+
+The library has been tested with following Python versions:
+
+    - [Python 3.4](https://www.python.org/download/releases/3.4.0/).
+    - [Python 3.5](https://www.python.org/download/releases/3.5.0/).
+    - [Python 3.6](https://www.python.org/download/releases/3.6.0/).
+    
+Once the required Python version is installed on system, the following requirements needs
+to be fulfilled to start building the library.
+
+- VirtualEnv:
+    ```
+    pip install virtualenv
+    ```
+    
+- PyBuilder:
+    ```
+    pip install pybuilder
+    ```
+        
+**NOTE** : The above commands will download the latest version of libraries, and have been tested against the recent versions. 
+
+Once above requirements are installed, use following commands to build library or building the docs.
+
+- Building Library: You can build the library yourself to test out the functionality of library. Refer to 
+[Building docs](BUILDING.md) for documentation on how to build the library.
+
+
+- Installing the Library:
+    
+    - To install library using Pip (It is not yet hosted):
+    
+    ```bash
+        # X is version number of JanusGraph Python client supported
+        pip install janusgraph_python=X
+    ```
+    
+    - To install library from tarball, once it is built:
+     
+     ```bash
+        pip install target/dist/janusgraph_python/dist/janusgraph_python-X.tar.gz
+        # X is the version number of library
+     ```
```

Comment:
Add a section on [contributing](https://github.com/JanusGraph/janusgraph/blob/master/CONTRIBUTING.md) and [community](https://github.com/JanusGraph/janusgraph/blob/master/README.md#community). You could reference back to the main JanusGraph repo (the janusgraph-dotnet PR does this).